cmd/liqo-controller-manager/main.go

```diff
@@ -166,6 +168,8 @@ func main() {
 	flag.BoolVar(&kubeletMetricsEnabled, "kubelet-metrics-enabled", false, "Enable the kubelet metrics endpoint")
 	flag.Var(&nodeExtraAnnotations, "node-extra-annotations", "Extra annotations to add to the Virtual Node")
 	flag.Var(&nodeExtraLabels, "node-extra-labels", "Extra labels to add to the Virtual Node")
+	flag.Var(&labelsNotReflected, "labels-not-reflected", "List of labels (key) that must not be reflected")
```

Comment:
What do you think about calling them something like `skip-label-reflection` or `ignore-label-reflection`?